framework/packages/abstract-sdk/src/apis/verify.rs

```diff
@@ -105,7 +105,7 @@ impl<'a, T: AccountVerification> AccountRegistry<'a, T> {
         }
     }
 
-    fn account_id(&self, maybe_core_contract_addr: &Addr) -> AbstractSdkResult<AccountId> {
+    pub fn account_id(&self, maybe_core_contract_addr: &Addr) -> AbstractSdkResult<AccountId> {
```

Comment:
Not sure why it was not exposed. @CyberHoward is there any downside exposing this?
I will add doc-comment if we are okay with that

Reply:
Nope should be good!